ajadriver/linux/Makefile

```diff
@@ -32,6 +32,9 @@ ifeq ($(NVIDIA_SRC_DIR),)
 endif
 ifeq ($(NVIDIA_SRC_DIR),)
 	NVIDIA_SRC_DIR := $(shell find /usr/src/linux-* -name nv-p2p.h 2>/dev/null|head -1|xargs dirname 2>/dev/null)
+endif
+ifeq ($(NVIDIA_SRC_DIR),)
+	NVIDIA_SRC_DIR := $(shell find /usr/src/nvidia/nvidia-oot/include/linux -name nv-p2p.h 2>/dev/null|head -1|xargs dirname 2>/dev/null)
```

Comment:
Wouldn't you achieve the same by changing the `nvidia-*` to just `nvidia*` above on line 30?

Reply:
Very good point. Thanks for catching that. I just pushed a new commit.